web/src/beta/features/Assets/AssetHooks/hooks.tsx

```diff
@@ -0,0 +1,174 @@
+import { useCallback, useState, useRef, useMemo, useEffect } from "react";
+
+import { Asset, SortType } from "@reearth/beta/features/Assets/types";
+import { autoFillPage, onScrollToBottom } from "@reearth/beta/utils/infinite-scroll";
+import { useAssetsFetcher } from "@reearth/services/api";
+import { Maybe, AssetSortType as GQLSortType } from "@reearth/services/gql";
+import { useT } from "@reearth/services/i18n";
+
+const assetsPerPage = 20;
+
+const enumTypeMapper: Partial<Record<GQLSortType, string>> = {
+  [GQLSortType.Date]: "date",
+  [GQLSortType.Name]: "name",
+  [GQLSortType.Size]: "size",
+};
+
+function toGQLEnum(val?: SortType) {
+  if (!val) return;
+  return (Object.keys(enumTypeMapper) as GQLSortType[]).find(k => enumTypeMapper[k] === val);
+}
```

Comment:
Is this duplicated? If so, please just use the function already there

Reply:
sorry but where the duplicated one , it is defined only once in this file